sms-spam-classifier/README.md

```diff
@@ -0,0 +1,56 @@
+# SPAM SMS CLASSIFIER
+
+This is a spam SMS classifier that uses a  machine learning model to detect whether a user input string is a spam SMS or not. The model is built using the scikit-learn machine learning library, specifically using the Naive Bayes algorithm and is trained on a dataset of 5572 text messages collected by the [UCI Machine Learning Repository
```

Comment:
"that uses a machine learning model"
there's an extra space character in between "a" and "machine"